### ISSUE TITLE
Enable HTTPS Secure Connector

### ISSUE THREAD

**Issue by ctdao** [deklab/dektalk#6]
Create a keystore and activate SSL for HTTPS connection

See more: http://documentation.telestax.com/connect/configuration/Restcomm%20-%20Enable%20HTTPS%20secure%20connector%20on%20JBoss%20AS%207%20or%20EAP%206.html

**Comment by ctdao** [comment]
Available at: https://dektalk.homenet.org:8443/olympus

**Comment by ctdao** [comment]
Currently not working with Firefox Browser. Needs to investigate.

**Comment by ctdao** [comment]
I have created a keystore and activated SSL for HTTPS connection. To create a trusted certificate, I used a free and open certificate authority: Let's Encrypt. This CA is trusted by all current browsers except Firefox. Unfortunately, Mozilla do not have Let's Encrypt in its trusted root certificate store. This means that Firefox will not accept the certificates, and will throw an error. Need help on this issue.

Server is available at: https://www.dektalk.ml/olympus
